tests/test_workflows.py

```diff
@@ -63,7 +63,7 @@ def test_get_workflow(self):
 
     def test_estimate_workflow_cost(self):
         ret = dict(workflows=[dict(workflowId=1)])
-        with mock.patch("terra_notebook_utils.workflows.get_submission", return_value=ret):
+        with mock.patch("terra_notebook_utils.workflows.get_all_workflows", return_value=ret):
```

Comment:
`terra_notebook_utils.workflows.get_submission` seems to still exist: https://github.com/DataBiosphere/terra-notebook-utils/blob/xbrianh-subworkflow-costs/terra_notebook_utils/workflows.py#L32

Should it still have a test?

Reply:
Yes. And it does! The test was not removed.